packages/control-property-editor/src/components/ThemeSelectorCallout.tsx

```diff
@@ -17,7 +17,7 @@ export type ThemeName = 'dark' | 'light' | 'high contrast';
 export function ThemeSelectorCallout(): ReactElement {
     const { t } = useTranslation();
     const [isCalloutVisible, setValue] = useState(false);
-    const theme = localStorage.getItem('theme') ?? 'dark';
+    const theme = localStorage.getItem('theme') ?? 'dark modern';
```

Comment:
Please use `useLocalStorage` hook instead, which correctly prefixes the keys.

We can't also just change the theme names, because if there is an existing theme stored in local storage, then we will end up with no theme variables loaded, because a theme with that name no longer exists. Switching to the prefixed key, should avoid this issue.